README.md

```diff
@@ -110,14 +119,32 @@ array([[ 1.,  1.  ],
        [ 1.,  4.  ]])
 ```
 
-For more examples check our [Jupyter notebook demos](https://github.com/h2oai/h2o4gpu/tree/master/examples/py/demos).
+To test your installation of the R package, try the following example that builds a simple [XGBoost](https://github.com/dmlc/xgboost) random forest classifier:
+
+``` r
+require(h2o4gpu)
```

Comment:
should be `library(h2o4gpu)`.  They do the same thing, but `require()` is designed only for use within other functions.